<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ccs::main::CornerRefinerConfigurationWidget</class>
 <widget class="QGroupBox" name="ccs::main::CornerRefinerConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>353</height>
   </rect>
  </property>
  <property name="title">
   <string>Refine &amp;using a corner detector</string>
  </property>
  <property name="checkable">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Half Window Size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="windowSizeWidthSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
       <property name="value">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="windowSizeHeightSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>256</number>
       </property>
       <property name="value">
        <number>11</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Zero Zone</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="zeroZoneWidthSpinBox">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="zeroZoneHeightSpinBox">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="ccs::main::TermCriteriaConfigurationWidget" name="termCriteriaConfigWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ccs::main::TermCriteriaConfigurationWidget</class>
   <extends>QWidget</extends>
   <header>main/TermCriteriaConfigurationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>windowSizeWidthSpinBox</tabstop>
  <tabstop>windowSizeHeightSpinBox</tabstop>
  <tabstop>zeroZoneWidthSpinBox</tabstop>
  <tabstop>zeroZoneHeightSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
